test/utils/test-utils.js

```diff
@@ -70,6 +70,12 @@ export function createFixture(modelName, fixture) {
   }
 }
 
+export async function createFixtures(fixtures) {
+  for (const model in fixtures) {
```

Comment:
this could be just:
```ts
await Promise.all(Object.entries(fixtures).map(([model, fixture]) => createFixture(model, fixture)))
```

Reply:
We need to create the fixtures in a particular order so that e.g. Participants are created before ParticipantDates, so the Promise.all() doesn't work. There could be more elegant approaches to the fixture creation though 🤔

Reply:
Oh right. This is a self-contained function so this for-loop isn't that bad really anyways. Using `Object.entries` would make the indexing a bit clleaner though:

```js
for (const [model, fixture] of Object.entries(fixtures)) {
  await createFixture(model, fixture)
}
```